base_images/fedora_base-setup.sh

```diff
@@ -16,6 +16,11 @@ REPO_DIRPATH=$(realpath "$SCRIPT_DIRPATH/../")
 # shellcheck source=./lib.sh
 source "$REPO_DIRPATH/lib.sh"
 
+# Switch to Rawhide
```

Comment:
@cevich I see there's a `shellcheck disable=SC2154` here. Do I need to move it before this part ?

Reply:
You need to disable SC2154 for any env. var. that's defined elsewhere, like `$CIRRUS_foo` for example.